package.json

```diff
@@ -127,8 +125,9 @@
     "lint-staged": "^12.2.2",
     "msw": "^0.36.3",
     "pptr-testing-library": "^0.7.0",
+    "sass": "^1.48.0",
+    "puppeteer": "^13.1.1",
```

Comment:
Can you revert this change too? It's going to cause a conflict later on, and we haven't updated the dependency version in the develop branch.

Reply:
Just puppeteer, I mean.

Reply:
And regenerate the yarn.lock accordingly.

Reply:
@0xJem  OK, done :)